Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.5
+FROM bitwalker/alpine-erlang
```

Comment:
@webdeb I have to say, I came to the same conclusion while testing around last night! The issue we had with this dockerfile was `WORKDIR ${HOME}` actually -- it's set in alpine-erlang, but not default alpine so it fails. Removing that and adding openssl & ncurses-libs makes it work. We had no issues with ERTS not being in the path though.

Also the way run and CMD are set up currently won't pass signals properly! That means graceful shutdown via `:init.stop` won't work.

I'll try and open a PR later this week, once our internal Dockerfile stabilizes.

Another issue is copying the whole release tarball, then later unpacking and deleting it -- it will be gone in the final image but show up as a docker layer (effectively doubling the file size), unless you flatten out the layers using the experimental `--squash` flag.